addons/godot-xr-tools/examples/fall_damage.gd

```diff
@@ -0,0 +1,86 @@
+@tool
```

Comment:
Should we move this into the demo? (possibly as a separate thing seeing we already did this in this way for Godot 3)